akka-http-core/src/main/scala/akka/http/impl/util/Rendering.scala

```diff
@@ -373,4 +373,4 @@ private[http] class CustomCharsetByteStringRendering(nioCharset: Charset, sizeHi
     }
     charBuffer.clear()
   }
-}
\ No newline at end of file
+}
```

Comment:
Strange formatting in github - does this file have strange line endings? Not really related to this PR though

Reply:
It has this line which is more than GitHub's Scala highlighter can manage:

```
val `\"` = CharPredicate('\\', '"')
```